src/pages/why-choose-us/_africa-forex-award.tsx

```diff
@@ -1,45 +1,166 @@
 import React from 'react'
-import styled from 'styled-components'
 import { graphql, useStaticQuery } from 'gatsby'
-import { QueryImage } from 'components/elements'
-import { WithIntl } from 'components/localization'
-import { Desktop, Mobile } from 'components/containers'
+import styled from 'styled-components'
+import { BackgroundImage, Header, QueryImage } from 'components/elements'
+import { WithIntl, Localize } from 'components/localization'
+import { Flex, Desktop, Mobile } from 'components/containers'
 
 const query = graphql`
     query {
-        africa_award: file(relativePath: { eq: "choose-us/africa_award.png" }) {
+        africa_award_background: file(relativePath: { eq: "choose-us/africa_award_bg.png" }) {
+            ...fadeIn
+        }
+        africa_award_background_mobile: file(
+            relativePath: { eq: "choose-us/africa_award_bg_mobile.png" }
+        ) {
             ...fadeIn
         }
-        africa_award_mobile: file(relativePath: { eq: "choose-us/africa_award_mobile.png" }) {
+        africa_award: file(relativePath: { eq: "choose-us/africa_award.png" }) {
             ...fadeIn
         }
     }
 `
 
-const ImgWrapper = styled.div`
+const AwardImageWrapper = styled.div`
     position: relative;
-    max-height: 1050px;
-    max-width: 1680px;
-    left: 50%;
-    transform: translateX(-50%);
+
+    /* right: -5.5%; */
+    top: 40px;
+    margin-top: 40px;
+    margin-right: -100px;
+    z-index: 2;
+    padding: 38px 62px;
+    max-width: 414px;
+    background: #ffffff;
+
+    /* box-shadow: 0px 32px 64px -12px rgba(14, 14, 14, 0.14); */
+    border-radius: 8px;
+    @media (max-width: 1400px) {
+        max-width: 35%;
+        margin-right: -6%;
+    }
+
+    @media (max-width: 1300px) {
+        margin-right: -3%;
+    }
+    @media (max-width: 1200px) {
+        padding: 19px 31px;
+        margin-right: 1%;
+    }
+
+    @media (max-width: 1100px) {
+        margin-right: 1%;
+        margin-top: 1%;
+    }
 `
-const StyledQueryImage = styled(QueryImage)`
-    .gatsby-image-wrapper [data-main-image] {
-        object-fit: contain !important;
+const AwardImageWrapperMobile = styled.div`
+    z-index: 2;
+    padding: 36px 18px;
+    max-width: 328px;
+    background: #ffffff;
+
+    /* box-shadow: 0px 32px 64px -12px rgba(14, 14, 14, .14); */
+
+    /* box-shadow: 0px 25px 50px -9.5px rgba(14, 14, 14, 0.14); */
+    border-radius: 8px;
+`
+
+const StyledFlex = styled(Flex)`
+    padding: 80px 120px 160px 120px;
+
+    @media (max-width: 1200px) {
+        padding: 0 50px 40px 50px;
+    }
+`
+const StyledFlexMobile = styled(Flex)`
+    /* padding: 80px 120px 160px 120px;
+    
+    @media (max-width: 1200px) {
+        padding: 0 50px 40px 50px;
+    } */
+`
```

Comment:
do we need this component? 
the whole style is commented.

Reply:
Forgot about that, thanks